AAOSL/Hops.agda

```diff
@@ -43,34 +44,58 @@ module AAOSL.Hops where
  -- The level of an index is 0 for index 0,
  -- otherwise, it is one plus the number of times
  -- that two divides said index.
- -- TODO-1: document reasons for this pragma and justify it
+ --
+ -- lvlOf must be marked terminating because in one branch
+ -- we make recursive call on the quotient of the argument, which
+ -- is not obviously smaller than that argument
```

Comment:
It would be nice to punctuate this (commas around "in one branch", period at the end) and ever so slightly more substantively, I would avoid judgment about what's "obvious".  You mean "obvious to Agda", but maybe "Agda cannot prove" is better?